lib/main.dart

```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
-             movie.thumbnail,
-           ),
-           Row( // Score and title
-             children: [
-               Padding(
-                 padding: const EdgeInsets.all(8.0),
-                 child: Card( // show the score
-                   shape: RoundedRectangleBorder(
-                     borderRadius: BorderRadius.circular(40), // made it circular
-                     side: const BorderSide(width: 5, color: Colors.indigo), // add a border
-                   ),
-                   color: Colors.blueAccent,
-                     child: Padding(
-                       padding: const EdgeInsets.all(15),
-                       child: Text(
-                         '${movie.voteAverage}',
-                         textScaleFactor: 2,
-                       )
-                     ),
-                 ),
-               ),
-               Expanded(
-                 child: Text( // Title
-                   movie.title,
-                   style: const TextStyle(
-                       color: Colors.white,
-                       fontWeight: FontWeight.bold,
-                       fontSize: 20.0
-                   ),
-                 ),
-               ),
-             ],
-           ),
-          SingleChildScrollView( // this permit can be scroll
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: Container(
+      color: Colors.black87,
+      child: Column(
+        children: <Widget>[
+          Image.asset(
+            // Top Image
+            movie.thumbnail,
+          ),
+          Row(
+            // Score and title
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Card(
+                  // show the score
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(80),
+                    // made it circular
+                    side: const BorderSide(
+                        width: 5, color: Colors.indigo), // add a border
+                  ),
+                  color: Colors.blueAccent,
+                  child: Padding(
+                      padding: const EdgeInsets.all(15),
+                      child: Text(
+                        '${movie.voteAverage}',
+                        textScaleFactor: 2,
+                      )),
+                ),
+              ),
+              Expanded(
+                child: Text(
+                  // Title
+                  movie.title,
+                  style: const TextStyle(
+                      color: Colors.white,
+                      fontWeight: FontWeight.bold,
+                      fontSize: 20.0),
+                ),
+              ),
+            ],
+          ),
+          SingleChildScrollView(
+            // this permit can be scroll
             scrollDirection: Axis.horizontal, // in the horizontal axis
             padding: const EdgeInsets.fromLTRB(10.0, 0.0, 10.0, 0.0),
```

Comment:
you can change this to EdgeInsets.symmetric(horizontal: 10.0)  and add 10.0 to constants